src/js/modal-dialog.js

```diff
@@ -454,17 +461,25 @@ class ModalDialog extends Component {
    * @listens keydown
    */
   handleKeyDown(event) {
+    const spatialNavigation = this.player_.spatialNavigation;
+    const isSpatialNavlistening = spatialNavigation && spatialNavigation.isListening && !spatialNavigation.isPaused;
 
-    // Do not allow keydowns to reach out of the modal dialog.
-    event.stopPropagation();
+    // Do not allow keydowns to reach out of the modal dialog unless spatialNavigation is enabled.
+    if (!isSpatialNavlistening) {
+      event.stopPropagation();
+    }
 
-    if (keycode.isEventKey(event, 'Escape') && this.closeable()) {
+    // If 'Esc' is pressed or Backspace is pressed & spatialNavigation is enabled & Modal is 'closeable'.
+    if (keycode.isEventKey(event, 'Escape') || (keycode.isEventKey(event, 'Backspace') && isSpatialNavlistening) && this.closeable()) {
```

Comment:
Should we ensure we only act on the `Escape` keycode here when spatial navigation is NOT enabled? Or do we want to close the modal no matter what if escape is hit?

Reply:
Hello, we want to close the modal no matter what if escape is hit, the way this line of code is written is to be not intrusive with the key presses  of the player that where already  working.